### PR TITLE
[terra-clinical] Drop intl v2-v4 support.

### PR DESCRIPTION
**Pull request by saket2403**
### Summary

The Following components will require MVB due to react-intl-v5 upgrade -

- Terra-clinical-data-grid
- Terra-clinical-onset-picker
- Terra-clinical-result

Closes #

### Deployment Link

https://terra-clinical-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-clinical-data-grid/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Breaking Changes
+  * Dropped v2-v4 react-intl support
```
Comment (supreethmr) [inline packages/terra-clinical-data-grid/CHANGELOG.md:6]: nit:  line break is needed between breaking changes and fixed section.

File: packages/terra-clinical-header/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
```
Comment (supreethmr) [inline packages/terra-clinical-header/CHANGELOG.md:5]: nit: need a line break here.

**Review by supreethmr — APPROVED** [review]

**Review by supreethmr — CHANGES_REQUESTED** [review]
Can you add `A11y-MVB` branch to travis : https://github.com/cerner/terra-clinical/blob/ac3cbf5e7cb69b786a9c0d26d7472764d98a9d32/.travis.yml#L7 

This would trigger travis build !!

File: packages/terra-clinical-detail-view/package.json [hunk 3]
```diff
@@ -40,9 +40,9 @@
     "precompile": "rm -rf lib",
     "test": "npm run lint && npm run jest && npm run wdio",
     "jest": "jest --config ../../jest.config.js",
-    "wdio-default": "cd ../.. && npx wdio",
-    "wdio-lowlight": "cd ../.. && THEME=clinical-lowlight-theme npx wdio",
-    "wdio-fusion": "cd ../.. && THEME=orion-fusion-theme npx wdio",
+    "wdio-default": "cd ../.. && terra wdio",
```
Comment (xenoworf) [inline packages/terra-clinical-detail-view/package.json:43]: I'm curious what difference does this change make?

File: packages/terra-clinical-data-grid/tests/jest/DataGrid.test.jsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 /* eslint-disable import/no-extraneous-dependencies */
-import { mountWithIntl } from 'terra-enzyme-intl';
+import { mountWithIntl } from '@cerner/terra-enzyme-intl';
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-clinical-data-grid/tests/jest/DataGrid.test.jsx:3]: Terra-enzyme-intl was changed to a scoped package in v4, Hence the change. [Upgrade Guide Link](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-enzyme-intl/UpgradeGuide.md) for the same

File: packages/terra-clinical-data-grid/tests/jest/__snapshots__/DataGrid.test.jsx.snap [hunk 5]
```diff
@@ -1933,33 +1933,6 @@ exports[`DataGrid Snapshots should render a DataGrid with selectable rows, colum
 
 exports[`getDerivedStateFromProps correctly applies the theme context className 1`] = `
 <ThemeContextProvider
-  intl={
```
Comment (xenoworf) [inline packages/terra-clinical-data-grid/tests/jest/__snapshots__/DataGrid.test.jsx.snap:1936]: Are such intl objects no longer needed, or are they still used but moved elsewhere?

**Review by xenoworf — APPROVED** [review]
Approved but of course please fix the build before merging. I'm glad for this change!

**Review by supreethmr — APPROVED** [review]

File: packages/terra-clinical-data-grid/tests/jest/__snapshots__/DataGrid.test.jsx.snap (hunk 5 above)
Comment (saket2403) [reply packages/terra-clinical-data-grid/tests/jest/__snapshots__/DataGrid.test.jsx.snap:1936]: The data grid [jests](https://github.com/cerner/terra-clinical/blob/fdd35b3a0dd7d4d7c443670d68488b776b69be22/packages/terra-clinical-data-grid/tests/jest/DataGrid.test.jsx#L388) are written using WrappedComponent. So the Data Grid is rendered without the injectIntl HOC in the snapshots.

**Pull request merged**